src/main/g8/default.properties

```diff
@@ -1,5 +1,5 @@
 name = My Udash Application
 package = io.company.app
-scala_version = 2.12.6
-udash_version = 0.7.1
-sbt_version = 1.1.6
+scala_version = 2.12.8
```

Comment:
`sbt new UdashFramework/udash.g8` seems to still propose 2.12.6 as default scala version. Is that desired behaviour?

Reply:
`sbt new UdashFramework/udash.g8` fetches master branch from this repo